server/server/src/account_service.rs

```diff
@@ -73,6 +73,7 @@ pub async fn new_account(
         tx.usernames.insert(username, owner);
         tx.owned_files.insert(owner, owned_files);
         tx.shared_files.insert(owner, HashSet::new());
+        tx.file_children.insert(*root.id(), HashSet::new());
```

Comment:
I was able to get the fuzzer to raise this issue which is not a big deal but does result in the server reporting an index integrity validation failure
```
Trial {
    id: 30603094-ec37-4c02-89c6-c5add18b5c17,
    target_clients: 2,
    target_steps: 2,
    steps: [
        SyncAndCheck,
    ],
    completed_steps: 1,
    status: Failed(
        "Server reported validation failures: AdminValidateServer {\n    users_with_validation_failures: {},\n    usernames_mapped_to_wrong_accounts: {},\n    usernames_mapped_to_nonexistent_accounts: {},\n    usernames_unmapped_to_accounts: {},\n    owners_mapped_to_unowned_files: {},\n    owners_mapped_to_nonexistent_files: {},\n    owners_unmapped_to_owned_files: {},\n    owners_unmapped: {},\n    sharees_mapped_to_unshared_files: {},\n    sharees_mapped_to_nonexistent_files: {},\n    sharees_mapped_for_owned_files: {},\n    sharees_unmapped_to_shared_files: {},\n    sharees_unmapped: {},\n    files_mapped_as_parent_to_non_children: {},\n    files_mapped_as_parent_to_nonexistent_children: {},\n    files_mapped_as_parent_to_self: {},\n    files_unmapped_as_parent_to_children: {},\n    files_unmapped_as_parent: {\n        7380ad6b-23e7-4063-84a5-1cb7b6defdcb,\n    },\n    sizes_mapped_for_files_without_hmac: {},\n    sizes_mapped_for_nonexistent_files: {},\n    sizes_unmapped_for_files_with_hmac: {},\n    files_with_hmacs_and_no_contents: {},\n}",
    ),
    start_time: Instant {
        t: 12180595179579,
    },
    end_time: Instant {
        t: 12181273594963,
    },
}
```